kani-driver/src/call_cbmc_viewer.rs

```diff
@@ -22,7 +22,8 @@ impl KaniSession {
         let results_filename = alter_extension(file, "results.xml");
         let property_filename = alter_extension(file, "property.xml");
 
-        self.record_temporary_files(&[&results_filename, &property_filename]);
```

Comment:
I needed to call this function and I had a vector of artifacts. For that, I had two options, create a new vector of references to the elements of the initial vector or just update the function signature.

Since the old signature was rather odd, its argument was slice of references, I decided to just update the function and it's previous usages.